lisp/comp/comp.l

```diff
@@ -1242,9 +1242,9 @@
 	   (setq o (merge-pathnames ".o" o)))
    (let ((name nil) (c nil) (form nil)
 	 (ins) (cccom)
-	 (file.c (merge-pathnames ".c" file))
-	 (file.h (merge-pathnames ".h" file))
-	 (file.q (merge-pathnames ".q" file))
+	 (file.c (if o (merge-pathnames ".c" file) (merge-pathnames ".c" file)))
```

Comment:
@k-okada this is....you mean `(file.c (if o (merge-pathnames ".c" o) (merge-pathnames ".c" file)))`?